backend/src/sync/mod.rs

```diff
@@ -74,6 +76,26 @@ pub(crate) struct SyncConfig {
     /// load on Opencast, increase to speed up download a bit.
     #[config(default = 8)]
     concurrent_download_tasks: u8,
+
+    /// List of deletion modes that which, if any, realm pages are to be deleted
+    /// automatically when the corresponding Opencast item (series, event or playlist)
+    /// is deleted.
```

Comment:
Something is wrong about this sentence I think? Is this just missing a "determine" before the "which"? Alternative suggestion:

> Enables auto deletion of realm pages whenever an Opencast item (event, series, playlist) is deleted.

Other than that, I like the description.